packages/database/prisma/schema.prisma

```diff
@@ -98,12 +106,25 @@ model Booking {
   updatedAt     DateTime      @updatedAt
   startDate     DateTime
   endDate       DateTime
-  bookingStatus BookingStatus @default(PENDING)
+  bookingStatus BookingStatus @default(WAITING_FOR_GUIDE_CONFIRMATION)
   postId        Int
   userId        Int
   guideId       Int
 
   post  Post  @relation(fields: [postId], references: [id])
   user  User  @relation(fields: [userId], references: [id])
   guide Guide @relation(fields: [guideId], references: [id])
+  Transaction Transaction[]
```

Comment:
Could you rename all list variable to be plural and uncapitalized (e.g. transactions) through the entire schema.
Those are my fault not reviewing thoroughly in previous sprint.